Gorilla_vs_Matlab.Rmd

```diff
@@ -0,0 +1,352 @@
+---
+title: "Experimental modalities - Gorilla versus Matlab"
+author: "Minah Chang"
+date: "1 March, 2022"
+output: html_document
+---
+
+### Data preparation
+
+From the data, "distractor trials" are discarded and not considered for further analyses in both experimental modalities.
+
+* All Distractor trials were removed (4186 values)
+
+```{r setup, include=FALSE}
+# Load and install (if missing) the recquired packages
+list.of.packages <- c("tidyverse", 
+                      "knitr", 
+                      "reshape2", 
+                      "lme4",
+                      "nlme",
+                      "car")
+new.packages <- list.of.packages[!(list.of.packages %in% installed.packages()[,"Package"])]
+if(length(new.packages)) install.packages(new.packages)
+
+lapply(list.of.packages, library, character.only = TRUE)
+
+# set script folder as working dir
+setwd(dirname(rstudioapi::getActiveDocumentContext()$path))
+
+# set to true to show the code chunk
+knitr::opts_chunk$set(echo = TRUE)
+```
+
+### Load processed and merged data
+
+The header column is organized as follows:
+
+* 1. pseudoid
+* 2. group (EB or SC)
+* 3. session (matlab or gorilla)
+* 4. category (integer 1, 3, 4, 6-9)
+    + category 1: size
+    + category 3: brightness
+    + category 4: loudness
+    + category 6: velocity
+    + category 7: movement
+    + category 8: space
+    + category 9: distance
+* 5. wordcode
+* 6. language
+* 7. stimulus
+* 8. meaning
+* 9. value (good or goodop)
+* 10. response
+* 11. grade (1 if correct, 0 if incorrect)
+
+*Native accuracy: given that some words in specific native languages groups can "sound" with the opposite meaning as presented in Tzeng (2017), we recompute the accuracy for those stimuli that in our prestudy were mapped systematically to the opposite meaning. For these subset of stimuli, we consider the response correct if it does not match the original meaning and incorrect if it does.
+
+```{r}
+
+allData<-read.csv("data/allData.csv", header = TRUE)
+
+```
+
+Check that all the participants have the same number of responses per category
+
+```{r}
+
+allData %>% 
+  group_by(pseudoid, session, group, category) %>% 
+  count() %>% 
+  pivot_wider(names_from = category, values_from = n) %>% 
+  print(n = 46)
+
+```
+
+### Analysis
+
+### Compute the raw and native accuracy for Gorilla and Matlab.
```

Comment:
Isn't this a repetition of accuracy computation in other scripts?

Reply:
> Isn't this a repetition of accuracy computation in other scripts?

Yes. Do we perform ANOVA or t-test here?